src/runtime/runtime_atsamd51g19.go

```diff
@@ -9,35 +9,35 @@ import (
 func initSERCOMClocks() {
 	// Turn on clock to SERCOM0 for UART0
 	sam.MCLK.APBAMASK.SetBits(sam.MCLK_APBAMASK_SERCOM0_)
-	sam.GCLK.PCHCTRL[7].Set((sam.GCLK_PCHCTRL_GEN_GCLK1 << sam.GCLK_PCHCTRL_GEN_Pos) |
+	sam.GCLK.PCHCTRL[sam.PCHCTRL_GCLK_SERCOM0_CORE].Set((sam.GCLK_PCHCTRL_GEN_GCLK1 << sam.GCLK_PCHCTRL_GEN_Pos) |
 		sam.GCLK_PCHCTRL_CHEN)
 
 	// sets the "slow" clock shared by all SERCOM
-	sam.GCLK.PCHCTRL[3].Set((sam.GCLK_PCHCTRL_GEN_GCLK1 << sam.GCLK_PCHCTRL_GEN_Pos) |
+	sam.GCLK.PCHCTRL[sam.PCHCTRL_GCLK_SDHC1_SLOW].Set((sam.GCLK_PCHCTRL_GEN_GCLK1 << sam.GCLK_PCHCTRL_GEN_Pos) |
```

Comment:
The value is 3, which is correct.
However, it might be better to name it something like GCLK_SERCOM0_SERCOM0_7_SLOW.

Reply:
The same goes for the other runtime_atsamd51*.go.

Reply:
runtime_atsamd51*.go is good except for this part.

Reply:
How about `PCHCTRL_GCLK_SERCOMX_SLOW`? I just added that to this PR.